parser/src/error.rs

```diff
@@ -10,8 +10,9 @@ pub enum Error {
 pub enum ParseError {
     InvalidInt(String),
     InvalidTraceCols(String),
-    MissingMainTraceCols(String),
     InvalidConst(String),
+    InvalidRandomValues(String),
+    MissingMainTraceCols(String),
     MissingBoundaryConstraint(String),
     MissingIntegrityConstraint(String),
```

Comment:
nit: If we are keeping the names in alphabetical order, then `InvalidInt` and `InvalidTraceCols` should probably be moved as well.